.travis.yml

```diff
@@ -28,6 +29,12 @@ matrix:
       python: "3.2"
     - env: DJANGO="Django>=1.10,<1.11"
       python: "3.3"
+    - env: DJANGO="Django>=1.11a,<2.0"
+      python: "3.2"
+    - env: DJANGO="Django>=1.11a,<2.0"
+      python: "3.3"
+    - env: DJANGO="https://github.com/django/django/archive/master.tar.gz"
+      python: "2.7"
```

Comment:
Django 2.0 doesn't support `pypy`. Maybe you can add it to exclude matrix in this PR.